.github/workflows/test.yml

```diff
@@ -16,8 +16,7 @@ on:
 
 env:
   environment: "qa"
-  timezone: "Europe/London"
-  node: ${{ vars.NODE_VERSION }}
```

Comment:
Just checking that we no longer need the `node` env var here

Reply:
Nope they have reffered directly now.